### ISSUE TITLE
Nuggets are not translated on YSOD

### ISSUE THREAD

**Issue by Rost54** [turquoiseowl/i18n#219]
http://i.imgur.com/8iiE68t.png

Any reasons not to handle Error event? http://i.imgur.com/w07hjDx.png

**Comment by turquoiseowl** [comment]
No, I don't think the issue has been raised before. I presumed error output would be caught as is.

 

A PR would be welcome if you have the time.
